ckanext/iati/theme/templates/home/index.html

```diff
@@ -38,8 +38,8 @@ <h2><a href="/using-iati-data">Using IATI data</a></h2>
         <a href="{{ h.url_for(controller='ckanext.iati.controllers.static:StaticController', action='dashboard') }}" class="block">Dashboard</a>
         <p>The IATI Dashboard provides statistics, charts and metrics on IATI data</p>
         <ul>
-          <li><a href="http://opendevtoolkit.org">See a list of projects, websites and tools that utilise IATI data</a></li>
-          <li><a href="/registry-api">Developer?  Resources to help.</a></li>
+          <li><a href="https://iatistandard.org/en/using-data/">Find out how use IATI data</a></li>
```

Comment:
> Find out how use IATI data

looks like this is missing the word “to”

Reply:
@andylolz does this mean that the link to opendevtoolkit.org does not make sense anymore?

Please also note that the /registry-api is now redirected to http://iatistandard.org/en/using-data/IATI-tools-and-resources/using-IATI-registry/ (check this https://github.com/IATI/ckanext-iati/blob/ckan-2.8.1/ckanext/iati/plugins.py#L126)

Reply:
> @andylolz does this mean that the link to [opendevtoolkit.org](http://opendevtoolkit.org) does not make sense anymore?

@derwas Yes, that’s correct – [it was decommissioned](https://discuss.iatistandard.org/t/open-development-toolkit-is-being-decommissioned-on-18-sept/1444), and [now redirects](http://opendevtoolkit.org) to iatistandard.org.